UIKitDZ/UIKitDZ/AnchorViewController.swift

```diff
@@ -0,0 +1,90 @@
+// AnchorViewController.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// AnchorViewController - страница с реализацией анкеров
+class AnchorViewController: UIViewController {
+    // MARK: Мшуц
+
+    let mainView = UIView()
+    let redLightView = LightView()
+    let yelloyLightView = LightView()
+    let greenLightView = LightView()
+
+    private var mainViewHeightConstraint: NSLayoutConstraint!
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setupUI()
+    }
+
+    // MARK: - При повороте смартфона
+
+    override func viewWillTransition(to size: CGSize, with coordinator: UIViewControllerTransitionCoordinator) {
+        super.viewWillTransition(to: size, with: coordinator)
+
+        coordinator.animate(alongsideTransition: { [weak self] _ in
+            guard let self = self else { return }
+
+            let isPortrait = size.height > size.width
+
+            if isPortrait {
+                self.mainViewHeightConstraint.constant = 376
+            } else {
+                self.mainViewHeightConstraint.constant = 316
+            }
+
+            self.view.layoutIfNeeded()
+        }, completion: nil)
+    }
+
+    // Установка визуала
+    private func setupUI() {
+        mainView.backgroundColor = .black
+        mainView.addSubview(redLightView)
+        mainView.addSubview(yelloyLightView)
+        mainView.addSubview(greenLightView)
+        view.addSubview(mainView)
+
+        redLightView.backgroundColor = .red
+        yelloyLightView.backgroundColor = .yellow
+        greenLightView.backgroundColor = .green
+
+        mainView.translatesAutoresizingMaskIntoConstraints = false
+        redLightView.translatesAutoresizingMaskIntoConstraints = false
+        yelloyLightView.translatesAutoresizingMaskIntoConstraints = false
+        greenLightView.translatesAutoresizingMaskIntoConstraints = false
+
+        setupStackView()
+    }
+
+    // Все анкера
+    func setupStackView() {
```

Comment:
```suggestion
    private func setupLayout() {
```
Емае)
Уже хотел писать что ты не то сделал. Чего у тебя этот метод так называется, если стека на экране нет?